src/test/java/seedu/address/logic/parser/GroupCommandParserTest.java

```diff
@@ -40,49 +42,62 @@ public void parse_invalidPreamble_failure() {
     @Test
     public void parse_allFieldsSpecified_success() {
         //Index targetIndex = INDEX_SECOND_PERSON;
-        NusId nusid = new NusId(VALID_NUSID_AMY);
-        String userInput = " id/" + nusid + GROUP_DESC_HUSBAND
-                + TAG_DESC_AMY;
+        NusId nusId = new NusId(VALID_NUSID_AMY);
+        Set<NusId> nusIdSet = new HashSet<>();
+        nusIdSet.add(nusId);
 
+        String userInput = " id/" + nusId + GROUP_DESC_HUSBAND + TAG_DESC_AMY;
```

Comment:
We didn't catch this earlier; but we should use the prefixes made in `logic.parser.CliSyntax` instead of adding it as a String itself